jazz_deployments/config/dev-config.json

```diff
@@ -0,0 +1,36 @@
+{
+  "DEPLOYMENT_TABLE": "{conf_stack_prefix}_Deployments_Dev",
+  "DEPLOYMENT_CREATION_REQUIRED_FIELDS": [
+    "service_id",
+    "service",
+    "domain",
+    "environment_logical_id",
+    "provider_build_url",
+    "provider_build_id",
+    "scm_commit_hash",
+    "scm_url",
+    "scm_branch",
+    "status",
+    "request_id"
+  ],
+  "REQUIRED_PARAMS": ["service", "domain", "environment"],
+  "OPTIONAL_PARAMS": ["status", "offset", "limit"],
+  "ARCHIVED_DEPLOYMENT_STATUS": "archived",
+  "DEPLOYMENTS_SORTING_KEY": "provider_build_id",
+  "DEPLOYMENTS_SORTING_ORDER": "desc",
+  "PAGINATION_DEFAULTS": {
+    "offset": 0,
+    "limit": 10,
+    "max_limit": 100
+  },
+  "DEPLOYMENT_STATUS": ["successful", "started", "failed", "archived", "aborted", "in_progress"],
+  "SVC_USER": "jobexec",
```

Comment:
Please drive these values from config (for reference: https://github.com/tmobile/jazz/pull/272)